src/ensembl/stories/shared-components/image-button/ImageButton.stories.tsx

```diff
@@ -1,45 +1,116 @@
+};
+
+trackPanelButtonStories.add(icon.imageName, () => {
+  const [buttonStatus, setVisible] = useState(Status.DEFAULT);
+  const computedStyles = { ...styles };
+
+  const toggleImage = () => {
+    switch (buttonStatus) {
+      case Status.DEFAULT:
+        return setVisible(Status.UNSELECTED);
+      case Status.UNSELECTED:
+        return setVisible(Status.SELECTED);
+      default:
+        return setVisible(Status.DEFAULT);
+    }
+  };
+  return (
+    <>
+      <div className={classNames(styles.containerStyles)}>
+        <div className={classNames(styles.imageCard)}>
+          <div className={classNames(styles.imageHolder)}>
+            <ImageButton
+              buttonStatus={buttonStatus}
+              description={'enable/disable'}
+              image={icon.image}
+              classNames={computedStyles}
+              onClick={toggleImage}
+            />
+          </div>
+          <div className={classNames(styles.imageDescription)}>
+            {buttonStatus}
+          </div>
+        </div>
+
+        <div className={classNames(styles.codeContent)}>
+          <h3>Usage:</h3>
+          {`import ImageButton, {ImageButtonStatus} from 'src/shared/image-button/ImageButton';`}
+          <br />
+          {`import { ReactComponent as ${icon.imageName} } from '${icon.imagePath}';`}
+          <br />
+          {`
+              <ImageButton buttonStatus={ImageButtonStatus.${buttonStatus.toUpperCase()}} 
+                description={'enable/disable'} 
+                image={${icon.imageName}}
+              />
+            `}
+        </div>
+      </div>
+      <div className={classNames(styles.containerStyles)}>
+        <div>Available styles:</div>
+        <div className={classNames(styles.imageCard)}>
+          <div className={classNames(styles.imageHolder)}>
+            <ImageButton
+              buttonStatus={Status.UNSELECTED}
+              description={'enable/disable'}
+              image={icon.image}
+              classNames={computedStyles}
+            />
+          </div>
+          <div className={classNames(styles.imageDescription)}>
+            {Status.UNSELECTED}
+          </div>
+        </div>
+
+        <div className={classNames(styles.imageCard)}>
+          <div className={classNames(styles.imageHolder)}>
+            <ImageButton
+              buttonStatus={Status.SELECTED}
+              description={'enable/disable'}
+              image={icon.image}
+              classNames={computedStyles}
+            />
+          </div>
+          <div className={classNames(styles.imageDescription)}>
+            {Status.SELECTED}
+          </div>
+        </div>
+
+        <div className={classNames(styles.imageCard)}>
+          <div className={classNames(styles.imageHolder)}>
+            <ImageButton
+              buttonStatus={Status.DISABLED}
+              description={'enable/disable'}
+              image={icon.image}
+              classNames={computedStyles}
+            />
```

Comment:
Do we still have the strikethrough sign over the disabled button? Are we using it anywhere?

Reply:
Not for ImageButton but we have it for VisibilityIcon

Reply:
I will leave it for now. We can decide later. It should be quick to remove it.

Reply:
Removed.